### DISCUSSION TITLE
'bokeh.server.django' is not a valid Python identifier

### DISCUSSION THREAD

**Discussion by TristanTTran** [bokeh/bokeh#11179] [General]
I was trying out the embedding of bokeh inside dajngo, as per this repo here https://github.com/bokeh/bokeh/tree/branch-2.4/examples/howto/server_embed/django_embed.

After extracting the code, installing django, bokeh, channels and panel as per instructed in Readme, and run django with _manage.py runserver 0.0.0.0:8000_, I got this error 
"_django.core.exceptions.ImproperlyConfigured: The app label 'bokeh.server.django' is not a valid Python identifier._"

Am I missing something in the installation?

**Comment by bryevdv** [comment]
This is a known issue https://github.com/bokeh/bokeh/issues/11148 and/or https://github.com/bokeh/bokeh/issues/10800

@TristanTTran The reality of the situation is that the group that funded the original addition of tight django / bokeh server integration have not funded its continuing maintenance, and this corner of Bokeh does not currently maintained. My only guidance is to stick to versions from around the orginal announcement of this  feature (roughly ~2.0.x and django 2.x) or else run a separate `bokeh serve` process and use the standard embedding APIs (e.g. `server_document`) for embedding from a separate `bokeh serve` process 

@mattpap should we turn off GH Discussions? I already spend a great deal of time supporting on Discourse and SO and do not want yet another channel to monitor.

**Reply by TristanTTran** [reply]
@bryevdv Thanks very much for the reply. Please know that we really appreciate the open-source and support you all have rendered. Cheers !

**Reply by mattpap** [reply]
> should we turn off GH Discussions? I already spend a great deal of time supporting on Discourse and SO and do not want yet another channel to monitor.

This has been discussed previously. GH discussions are intended for design discussions, as their format is better than issues for this purpose, and they are closer to the repository than discourse. There's a pinned thread that explains the purpose of this and points submitters to discourse for content like this.
